### ISSUE TITLE
[BUG] Type of mongoose-lean-defaults is not assignable to parameter type of Schema.plugin

### ISSUE THREAD

**Issue by lutherman** [DouglasGabr/mongoose-lean-defaults#20]
Hi,

The type of mongoose-lean-defaults **2.0.0** seems not assignable to type of function Schema.plugin() when using typescript.

I have a test codes like that:
```typescript
import { Model, Schema } from 'mongoose';
import * as mongooseLeanDefaults from 'mongoose-lean-defaults';

interface ITest {
  name: string;
}

const testSchema = new Schema<ITest, Model<ITest>>({
  name: {
    type: String,
    required: true,
    default: 'default',
  },
});

testSchema.plugin(mongooseLeanDefaults); // <-- error here
```

And its error message:
```
Argument of type 'typeof import("/Users/.../node_modules/mongoose-lean-defaults/index")' is not assignable to parameter of type '(schema: Schema<ITest, Model<ITest, {}, {}>, undefined>, opts?: any) => void'.
  Type 'typeof import("/Users/.../node_modules/mongoose-lean-defaults/index")' provides no match for the signature '(schema: Schema<ITest, Model<ITest, {}, {}>, undefined>, opts?: any): void'.ts(2345)
```

Regards,
Luther MAN

------------------------------
nodejs: 12.19.0
mongodb: 4.2.7
mongoose: 5.12.13
mongoose-lean-defaults: 2.0.0

**Comment by DouglasGabr** [comment]
Thanks for the issue, I'll fix it ASAP

**Comment by DouglasGabr** [comment]
Fixed in v2.0.1.
The import should be `import mongooseLeanDefaults from 'mongoose-lean-defaults';` tho, make sure to double check that

**Comment by lutherman** [comment]
Thank you for the quick fix.
